rustls-cert-gen/src/cert/mod.rs

```diff
@@ -0,0 +1,44 @@
+use rcgen::{Certificate, CertificateParams, Error};
+use std::{fs::File, io, path::Path};
+
+mod params;
+pub use params::BuildParams;
+
+mod ca;
+mod entity;
+mod signature;
+
+pub trait CertParams {
+	fn params(&self) -> &CertificateParams;
+}
+
+#[derive(Debug, Clone)]
+/// Struct to represent a Serialized Cert/Key pair
+pub struct SerializedEntity {
```

Comment:
I think we could come up with a better name for this. In Rustls-land we call something like this a [CertifiedKey](https://docs.rs/rustls/latest/rustls/sign/struct.CertifiedKey.html).

Maybe `PemCertifiedKey` with rustdoc explaining it as a PEM serialized Certificate and its corresponding PEM serialized private key.